### ISSUE TITLE
Scrollable region not keyboard accessible

### ISSUE THREAD

**Issue by joe-watkins** [joe-watkins/kfdb#12]
Ensure elements that have scrollable content are accessible by keyboard

[more information Link opens in a new window](https://dequeuniversity.com/rules/axe/4.10/scrollable-region-focusable?application=AxeChrome)
Element Location:
.overflow-y-auto
<div class="flex-grow p-4 overflow-y-auto scrollbar-hide" role="log" aria-live="polite">
To solve this problem, you need to fix at least (1) of the following:
Element should have focusable content
Element should be focusable

How to fix:
- Add tabindex="0" to the scrollable container
- Remove the "scrollbar-hide" class
- Add a min height of 480px to the container which will show scrollbar after it extends beyond that size
- Add aria-label="AI Assistant"

<img width="1436" height="751" alt="Image" src="https://github.com/user-attachments/assets/c15e0bf2-b05b-49b9-8c4f-eb4dda64ce45" />
